s3fs-fuse.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: s3fs-fuse
+  version: "1.94"
+  epoch: 0
+  description: FUSE-based file system backed by Amazon S3
+  copyright:
+    - license: GPL-2.0-only
+  # https://github.com/s3fs-fuse/s3fs-fuse/blob/master/COMPILATION.md#note---the-required-librariescomponents-required-to-run-s3fs-are
+  dependencies:
+    runtime:
+      - fuse2
+      - openssl
+      - curl
+      - libxml2
```

Comment:
I expect all of these to have soname depends; which are better than fixed depends. As soname depends will remain correct during transitions. Please remove these.